lab_2/find_lcs_length_optimized_test.py

```diff
@@ -4,17 +4,75 @@
 
 import timeit
 import unittest
-from lab_2.main import find_lcs_length, find_lcs_length_optimized
+from lab_2.main import find_lcs_length, find_lcs_length_optimized, tokenize_big_file
+from lab_2.tokenizer import tokenize
+from memory_profiler import memory_usage
 
 
 class FindLcsOptimizedTest(unittest.TestCase):
     """
     Checks for find_lcs_optimized function
     """
 
+    def test_find_lcs_length_optimized_ideal_case(self):
+        """
+        Tests that find_lcs_length_optimized
+            works just fine and not fails with big text
+        """
+        sentence_tokens_first_text = tokenize_big_file('lab_2/data.txt')
+        sentence_tokens_second_text = tokenize_big_file('lab_2/data_2.txt')
+        plagiarism_threshold = 0.3
+        actual = find_lcs_length_optimized(tuple(sentence_tokens_first_text),
+                                           tuple(sentence_tokens_second_text),
+                                           plagiarism_threshold)
+        self.assertTrue(actual)
+
+    def test_find_lcs_length_optimized_quickest_time(self):
+        """
+        Tests that find_lcs_length_optimized
+            works faster than  time reference
+        """
+        reference = 9.101999999927557e-06 + (9.101999999927557e-06 * 0.1)
```

Comment:
So students would not be able to test it locally, only in CI?

Reply:
yes, because this reference value is made using our CI configuration. Our computers processes data in a different way